tensilelite/Tensile/KernelWriterAssembly.py

```diff
@@ -9067,6 +9093,21 @@ def globalWriteElements(self, kernel, tPA, tPB, vectorWidths_2, vectorWidths_1,
           ssslist.append("Bias")
           useSize.append(True)
 
+      if vectorDataTypes.isValid() and (not isLdsLoaded):
+        factorDim = 0 # Currently no factorDim for scaleA, scaleB
```

Comment:
it is better to reject solution if user tried to set FactorDim != 0

Reply:
scaleA is always 0 and scaleB is always 1, so I set to 0.